### PR TITLE
Preflight - check ports

### PR DESCRIPTION
**Pull request by madirey**
Checks that critical ports are available on install:
- 3000
- 8000

https://docs.sublimesecurity.com/docs/quickstart-docker#troubleshooting
image

### CODE REVIEW

File: preflight_checks.sh [hunk 1]
```diff
@@ -58,6 +69,11 @@ if [ "$machine" == "linux" ]; then
     fi
 fi
 
+check_port 3000
+check_port 5432
+check_port 6379
```
Comment (cameron-dunn-sublime) [inline preflight_checks.sh:74]: Do `5432` and `6379` need to be available on the host? We don't expose these ports (note that this does differ between our `dev` and `main` branch -- on `dev` they are exposed).

Comment (madirey) [reply preflight_checks.sh:74]: Good question, I was wondering that myself. I suppose we don't need to if it's something customers won't run into.

Comment (cameron-dunn-sublime) [reply preflight_checks.sh]: I _think_ we won't need them, but I'm not a docker network expert so I do think it's worth testing. 

If ports only exposed internally are exposed then no, there shouldn't be any others. If all ports used within the platform need be available on the host, then there are a few: `8110`, `8100`, and I think both `strelka-coordinator` and `strelka-frontend` would have a port.

Comment (madirey) [reply preflight_checks.sh]: Yeah, I think this should only be affected by forwarded ports.

Comment (madirey) [reply preflight_checks.sh]: Got it down to just 3000 and 8000.

**Review by cameron-dunn-sublime — APPROVED** [review]

**Pull request merged**